upgrade/upgrade-scripts/execute

```diff
@@ -186,11 +186,26 @@ if [[ -n "$CURRENT_VERSION" ]]; then
 		"$ROOTFS_CONTAINER" "$ROOTFS_CONTAINER@execute-upgrade.$UNIQUE"
 fi
 
+#
+# Delete the central /etc/apt/sources.list and also all files in
+# /etc/apt/sources.list.d/ because only the Delphix repository
+# from the upgrade image must be used to install/upgrade packages.
+#
 if [[ -f /etc/apt/sources.list ]]; then
-	mv /etc/apt/sources.list /etc/apt/sources.list.orig ||
-		die "failed to save /etc/apt/sources.list"
+	rm /etc/apt/sources.list || die "failed to delete /etc/apt/sources.list"
+fi
+
+if [[ -d /etc/apt/sources.list.d ]]; then
+	for file in /etc/apt/sources.list.d/*; do
+		rm "$file" || die "failed to delete $file"
```

Comment:
If the directory has no files in it, then this will try to remove a file called "/etc/apt/sources.list.d/*". Example:
```
[root@ip-10-110-215-186 ~]# mkdir empty
[root@ip-10-110-215-186 ~]# for f in empty/*; do
> rm $f
> done
rm: cannot remove 'empty/*': No such file or directory
```
Fwiw, this is the same behavior if the directory itself doesn't exist, so the check at line 198 doesn't do anything. For example:
```
[root@ip-10-110-215-186 ~]# for f in /non/existent/directory/*; do
> rm $f
> done
rm: cannot remove '/non/existent/directory/*': No such file or directory
```

I think it would be simple enough to do all of this without checking and without any loops:
```
rm -f /etc/apt/sources.list
rm -f /etc/apt/sources.list.d/*
```